.github/workflows/lint.yml

```diff
@@ -37,3 +37,12 @@ jobs:
 
       - name: Check idiomatic code
         run: cargo clippy --all --all-features -- -D warnings
+
+      - name: Check for broken Markdown links
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with:
+          use-quiet-mode: 'yes'
+          use-verbose-mode: 'yes'
```

Comment:
this combo seems weird at first, but what it does is:

- only show us errors that occur instead of successful links AND errors
- output full HTTP info for broken links

this config seems to be popular among other projects using this tool